.gitignore

```diff
@@ -19,4 +19,11 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
-project.xcworkspace
\ No newline at end of file
+project.xcworkspace
```

Comment:
Why do the project files need to be ignored? How does this affect development with a clean clone of the repo?

Reply:
I am not sure, let me check again